ostrich_swmm/inject.py

```diff
@@ -320,6 +417,21 @@ def inject_parameters_into_input(input_parameters, input_template):
             'values': lid_values,
             'comment': None,
         })
+        
+            
+    with open('excess_rb.csv', 'wb') as outcsv:
+        writer = csv.writer(outcsv)
+        writer.writerow(excess_colnames)
+        writer.writerow(excess_rb)
+        writer.writerow(nlid)
+    with open('summary.txt', 'w') as summary:
+        summary.write("Total Number of Rain Barrels =" + str(sum(nlid))+ '\n')
+        summary.write("Total Number of RB1 =" +str(sum(nlid[0:len(nlid):2]))+ '\n')
+        summary.write("Total Number of RB2 =" +str(sum(nlid[1:len(nlid):2]))+ '\n')
```

Comment:
There may be any number of barrel types using any kind of naming scheme in an input file. Looping over the LID types and writing one line per type would allow any scheme to be reflected in the summary file.

Reply:
Good point, that will help as we add more lids